packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -350,7 +356,11 @@ export class KeybindingWidget extends ReactWidget {
     protected renderRow(item: KeybindingItem, index: number): React.ReactNode {
         const { command, keybinding } = item;
         // TODO get rid of array functions in event handlers
-        return <tr className='kb-item-row' key={index} onDoubleClick={() => this.editKeybinding(item)}>
+        return <tr
+            className='kb-item-row'
+            key={index}
+            onDoubleClick={this.editKeybinding(item)}
+            onContextMenu={this.handleContextMenu(item)}>
```

Comment:
it does not make a different, a new instance is created on each call of `renderRow`.

A function should be created only once  and a pointer to it should be kept as a property.